### ISSUE TITLE
Gerrit 2.11 stable branch needed

### ISSUE THREAD

**Issue by lucamilanesio** [criteo/automerge-plugin#3]
Current automerge-plugin master isn't building with Gerrit 2.12 (master) because still uses the Gerrit 2.11 APIs.

We should have:
- stable-2.11 branch
- master branch with required API changes

I've forked the project on GerritForge/automerge-plugin with the stable-2.11 branch as GitHub does not allow the create a "branch create" request :-)

**Comment by lucamilanesio** [comment]
@debanne @dopuskh3 why don't we migrate this plugin to gerrit-review.googlesource.com? Every proposed change will be automatically published and advertised in the Gerrit home page and will encourage more contributors to give their input :-)

**Comment by debanne** [comment]
I created the stable-2.11 branch. What would replace the travis builds if we move to gerrit-review.googlesource.com? Do you have a pointer about the process to add a new plugin in gerrit-review?

**Comment by lucamilanesio** [comment]
@debanne yes, it is https://gerrit-ci.gerritforge.com, the YAML file to define a job is quite simple (e.g. https://gerrit.googlesource.com/gerrit-ci-scripts/+/refs/heads/master/jenkins/gerrit-buck-plugin-admin-console.yaml). Much simpler than Travis ;-)

**Comment by lucamilanesio** [comment]
@debanne any feedback? Would you like to move to gerrit-review.googlesource.com?

**Comment by debanne** [comment]
Hi Luca, yes it looks like a very good idea :)

**Comment by lucamilanesio** [comment]
I requested to the Gerrit mailing list to move the project there: see https://groups.google.com/forum/#!topic/repo-discuss/iRQY41fF35s
